src/cephadm/cephadm

```diff
@@ -6867,6 +6867,13 @@ def command_rm_daemon(ctx):
     else:
         call_throws(ctx, ['rm', '-rf', data_dir])
 
+    if 'tcp_ports' in ctx and ctx.tcp_ports is not None:
+        ports: List[int] = list(map(int, ctx.tcp_ports.split()))
```

Comment:
Why not a list comprehension: `[int(p) for p in ctx.tcp_ports.split()]`?  I think list comprehensions are bit more idiomatic for python.

Reply:
That's right :+1:  Fixed.